.github/workflows/generate-docs.yml

```diff
@@ -0,0 +1,44 @@
+name: Generate docs
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  generate-docs:
+    name: generate-docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 14.15.0
+
+      - name: Install Dependencies
+        run: |
+          yarn example && yarn
+
+      - name: Build docs
+        run: yarn docs
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
```

Comment:
I think we should add `force_orphan: true` there isn't a need to keep a version history for docs.